.github/workflows/package-dependencies-update.yml

```diff
@@ -29,58 +29,61 @@ jobs:
         run: |
           npm install
 
       - name: Verify outdated node modules package dependencies versions
-        continue-on-error: true # Continuing on error, we need to check for outdated modules, so it's okay for this step to fail.
+        continue-on-error: true # Continuing on error. The initial command will return an exit 1 code; however, this is okay.
         run: |
           echo "Preparing to list outdated node modules versions within current build"
           npm outdated
 
       - name: Update node modules package dependencies versions within package.json
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         run: |
+          mkdir -p .github/logs
           echo "Preparing to update all outdated package modules to latest versions"
-          npx npm-check-updates -u
+          echo "Ignoring package update for cypress"
+          npx npm-check-updates -u -x cypress >> .github/logs/update-log-$(date +%m-%d-%y).txt
 
       - name: Installing updated node modules package dependencies within the project
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         run: |
           echo "Preparing to install the updated package dependencies"
           npm install
 
       - name: Resolve node modules package vulnerabilities
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
+        continue-on-error: true # Continuing on error. This step is to avoid vulnerabilities; however, we incase it fails we will need to resolve them manually.
         run: |
           echo "Preparing to address security vulnerabilities in the updated package dependencies."
           npm audit fix
 
-      - name: Create report to README
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
+      - name: Add updated log report to README # Once the PR has been merged, the link will be active and available for view.
         run: |
-          echo -e "\nCreated automated PR: $(date +%S-%d-%m-%y)" >> README.md
+          echo -e "\nGenerated auto rebuild. View log file created: [update-log-$(date +%m-%d-%y)](https://github.com/stolostron/search-e2e-test/blob/main/.github/logs/update-log-$(date +%m-%d-%y).txt)" >> README.md
+
+      - name: Execute code linting # Executing this step incase the file format needs to be fixed.
+        run: |
+          npm run lint:fix
           git status
 
       - name: Create pull request for updated node modules versions
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         uses: peter-evans/create-pull-request@v3
         with:
           add-paths: .
-          body: >
-            This PR is auto-generated by the Git action [search-e2e-test](https://github.com/stolostron/search-e2e-test/blob/main/.github/workflows/package-dependencies-update.yml).
+          body: |
```

Comment:
Based upon the slack conversation, a good enhancement would be to display the actual packages that were outdated within the PR body. I haven't found a good way to capture those details yet within the body; however, this will be something to add in the future.

At this current moment, I am capturing the outdated packages within a log file.